runtime: command "make test" and "make coverage" can excute, but exit with non-zero status code

# Description of problem

excute `make test` and `make coverage` under the project dir "src/runtime"

# Expected result

command exit with zero status code

# Actual result

command exit with non-zero status code, like 
```sh
$ make test
# github.com/kata-containers/kata-containers/src/runtime/cli/containerd-shim-kata-v2
cli/containerd-shim-kata-v2/main.go:25:71: undefined: project
cli/containerd-shim-kata-v2/main.go:25:110: undefined: version
cli/containerd-shim-kata-v2/main.go:25:119: undefined: commit
...
make: *** [Makefile:614: go-test] Error 2
```

--- a/src/runtime/Makefile
+++ b/src/runtime/Makefile
@@ -610,6 +610,7 @@ ifeq ($(shell id -u), 0)
 endif
 
 go-test: $(GENERATED_FILES)
+	$(QUIET_BUILD)(cd $(SHIMV2_DIR)/ && ln -fs $(GENERATED_CONFIG))
 	go test -v -mod=vendor ./...
 
 check-go-static:
@@ -617,6 +618,7 @@ check-go-static:
 	$(QUIET_CHECK)../../ci/go-no-os-exit.sh ./virtcontainers
 
 coverage:
+	$(QUIET_BUILD)(cd $(SHIMV2_DIR)/ && ln -fs $(GENERATED_CONFIG))
 	go test -v -mod=vendor -covermode=atomic -coverprofile=coverage.txt ./...
 	go tool cover -html=coverage.txt -o coverage.html
 

